include/mbedtls/ecp.h

```diff
@@ -276,15 +276,15 @@ mbedtls_ecp_group;
 
 #if !defined(MBEDTLS_ECP_FIXED_POINT_OPTIM)
 /*
- * Trade memory for speed on fixed-point multiplication.
+ * Trade ROM usage for speed on fixed-point multiplication.
  *
  * This speeds up repeated multiplication of the generator (that is, the
  * multiplication in ECDSA signatures, and half of the multiplications in
  * ECDSA verification and ECDHE) by a factor roughly 3 to 4.
  *
- * The cost is increasing EC peak memory usage by a factor roughly 2.
+ * The cost is increasing ROM usage by a factor roughly 2.
```

Comment:
I don't think this sentence makes that much sense for code size. Perhaps we can give some precise numbers here, for example: "For each n-bit Short Weierstrass curve that is enabled, this adds 4n bytes of code size if n < 384 and 8n otherwise."